Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1483,7 +1509,9 @@ - (void)serverCallback:(BNCServerResponse *)response {
             } else {
                 [BNCPreferenceHelper setSessionParams:NO_STRING_VALUE];
             }
-            
+            if ([BNCPreferenceHelper getNeedAppListCheck]) {
```

Comment:
I'm not super familiar with the flow of the Branch stuff, but does this same check need to be repeated in a bunch of places? Seems like it'd be best to just do it once somewhere after init has happened.